test/monaco/monaco.test.ts

```diff
@@ -135,4 +137,126 @@ describe('API Integration Tests', function (): void {
 			'\t\t\'\'\'Make the monkey eat N bananas!\'\'\''
 		]);
 	});
+	describe('Accessibility', function (): void {
+		beforeEach(async () => {
+			await page.goto(APP);
+		});
+
+		it('Should not have critical accessibility violations', async () => {
+			await page.evaluate(`
+		(function () {
+			instance.focus();
+			instance.trigger('keyboard', 'cursorHome');
+			instance.trigger('keyboard', 'type', {
+				text: 'a'
+			});
+		})()
+		`);
```

Comment:
You can strongly type these so you're not working with strings, the xterm.js tests show this in a simpler way to do this as the vscode ones are quite complex and need to go via Electron.

This proxy object shows you how they're written really nicely in xterm.js:
 https://github.com/xtermjs/xterm.js/blob/e9c547c1c6b67e9f09c24ccc007e19305f536e60/test/playwright/TestUtils.ts#L133

Reply:
Makes sense, but I'm not doing much with those. Just opening an editor.